packages/ui/package.json

```
@@ -32,6 +34,8 @@
     "vaul": "^1.1.2"
   },
   "devDependencies": {
+    "@types/react": "^19.2.2",
```

Biome (2.1.2)

[error] The key @types/react was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)